Simplified/AxisService/NYPLAxisLicenseService.swift

```diff
@@ -190,22 +189,21 @@ class NYPLAxisLicenseService: NYPLAxisLicenseHandling {
       let encryptedContent = encryptionNode[axisKeysProvider.contentKey] as? [String: Any],
       let encryptedContentValue = encryptedContent[axisKeysProvider.encryptedValue] as? String
     else {
-      logLicenseErrorAndLeave(
+      logLicenseErrorAndTerminateDownload(
         summary: NYPLAxisLicenseService.extractAESKeyFailureSummary,
         reason: NYPLAxisLicenseService.misssingAESKey)
       return nil
     }
     
     defer {
-      axisItemDownloader.dispatchGroup.leave()
+      taskSynchnorizer.endSynchronizedTask()
     }
     
     return Data(base64Encoded: encryptedContentValue)
   }
   
-  func logLicenseErrorAndLeave(summary: String,
-                               reason: String,
-                               additionalInfo: [String: String] = [:]) {
+  func logLicenseErrorAndTerminateDownload(
+    summary: String, reason: String, additionalInfo: [String: String] = [:]) {
```

Comment:
I think it's easier to read with the indentation like before (each parameter on a new line), just a suggestion